### PR TITLE
Add and load Lenna texture

### LINKED ISSUE #60: Add image texture to the cube
Add Lenna texture to cube primitive: https://en.wikipedia.org/wiki/Lenna

### PR DESCRIPTION
**Pull request by jowie94**
closes #60

### CODE REVIEW

File: Equinox/Equinox.vcxproj [hunk 1]
```diff
@@ -41,10 +41,10 @@
   </ImportGroup>
   <PropertyGroup Label="UserMacros" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">
-    <IncludePath>$(SolutionDir)External;$(IncludePath)</IncludePath>
+    <IncludePath>$(SolutionDir)External;$(SolutionDir)External\DevIL\include;$(IncludePath)</IncludePath>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|Win32'">
-    <IncludePath>$(SolutionDir)External;$(IncludePath)</IncludePath>
+    <IncludePath>$(SolutionDir)External;$(SolutionDir)External\DevIL\include;$(IncludePath)</IncludePath>
   </PropertyGroup>
```
Comment (polvila) [inline Equinox/Equinox.vcxproj:48]: Wasn't it included?

**Review by theepsi — APPROVED** [review]
Nice, I approve the changes! but I have a question, it will be cool to have 2 cubes? one with the procedural texture and another one with Lena?

File: Equinox/Equinox.vcxproj (hunk 1 above)
Comment (jowie94) [reply Equinox/Equinox.vcxproj:48]: Well not exactly, without this, the DevIL includes don't work

**Comment by jowie94** [comment]
@theepsi okey, I'll add two cubes while fixing merge conflicts

**Review by polvila — APPROVED** [review]

**Pull request merged**